homeassistant/components/light/nanoleaf.py

```diff
@@ -187,9 +195,15 @@ def turn_off(self, **kwargs):
 
     def update(self):
         """Fetch new state data for this light."""
-        self._brightness = self._light.brightness
-        self._color_temp = self._light.color_temperature
-        self._effect = self._light.effect
-        self._effects_list = self._light.effects
-        self._hs_color = self._light.hue, self._light.saturation
-        self._state = self._light.on
+        try:
+            self._available = self._light.available
+            self._brightness = self._light.brightness
+            self._color_temp = self._light.color_temperature
+            self._effect = self._light.effect
+            self._effects_list = self._light.effects
+            self._hs_color = self._light.hue, self._light.saturation
+            self._state = self._light.on
+        except Exception as err:  # pylint:disable=broad-except
```

Comment:
Why do we need the broad exception catching? If we expect an error here, we should be able to tell what it will be, right?